src/audio/dai.c

```diff
@@ -596,7 +604,7 @@ static int dai_check_for_xrun(struct comp_dev *dev, uint32_t copy_bytes)
 		return 0;
 
 	/* no data yet, we're just starting */
-	if (!dev->position)
+	if (dd->started)
```

Comment:
Is the comment above still necessary with this change?

Reply:
@paulstelian97 I thought about it, but in the same time this comment is still valid.